### PR TITLE
CLI delete-node: correction of handling an exception occurred during execution when not enough rights are on

### PR DESCRIPTION
**Pull request by pjanouse**
SSIA

### CODE REVIEW

File: core/src/main/java/hudson/cli/DeleteNodeCommand.java [hunk 1]
```diff
@@ -78,7 +78,7 @@ protected int run() throws Exception {
                 }
 
                 node.toComputer().doDoDelete();
-            } catch (AccessDeniedException e) {
+            } catch (AccessDeniedException2 e) {
```
Comment (olivergondza) [inline core/src/main/java/hudson/cli/DeleteNodeCommand.java]: It should be `org.acegisecurity.AccessDeniedException` IMO.

Comment (pjanouse) [inline core/src/main/java/hudson/cli/DeleteNodeCommand.java]: @olivergondza 
e.toString() says:
hudson.security.AccessDeniedException2: user is missing the Slave/Delete permission

Note: I caught that during the work on connect-node (PR#1923).

**Comment by oleg-nenashev** [comment]
Maybe makes sense to catch both eceptions

File: core/src/main/java/hudson/cli/DeleteNodeCommand.java (hunk 1 above)
Comment (olivergondza) [inline core/src/main/java/hudson/cli/DeleteNodeCommand.java]: Unfortunately, Jenkins is not very consistent in using its own subclass (`AccessDeniedException2`). Despite I am not sure if good old `AccessDeniedException` can bubble up here somehow, I do not see a reason why to be overly specific when catching exceptions.

**Comment by pjanouse** [comment]
Import corrected as suggested.

**Comment by olivergondza** [comment]
:+1:

**Comment by daniel-beck** [comment]
:+1:

Please make an effort to provide a useful commit message and PR description in future PRs.

**Comment by pjanouse** [comment]
Improved.

**Pull request merged**